test/linalg1.jl

```diff
@@ -2,7 +2,7 @@ debug = false
 
 import Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
-n = 10
+n = 10  # WARNING: n must be >= 10 for GeneralizedSchur tests
```

Comment:
Ah. It would be great if you could change the that to something like `1:div(n,2)` and `div(n,2)+1:n`.